.travis.yml

```diff
@@ -3,4 +3,6 @@ node_js:
   - "5.7"
   - "stable"
 before_script:
-  - npm install grunt-cli -g
+  - npm install -g gulp
+script:
+  - gulp test
```

Comment:
actually it's: delete the last two lines. There is a reason why Travis runs `npm test` out of the box